README.md

```diff
@@ -2,18 +2,150 @@
 Catalog for ESA data transfer study
 
 
-# Parsing and SQL
-### Describes the requirements for parsing and SQL
-sqlalchemy - Load Pandas dataframe into PostgreSQL database\
-pandas - create and parse csvs into dataframes\
-psycopg2 - Needed for compatibility with PostgreSQL in sqlalchemy\
+# Setup
 
-`pip3 install sqlalchemy`\
-`pip3 install pandas`\
-`pip3 install psycopq2`
+## Dependency
 
-# Django Python
---
+System dependency of `postgresql`.
 
-# APIs
---
+For python-specific dependency, see `requirements.txt`.
+
+## Installation
+
+- Install postgresql
+- create a virtualenv : `python -m venv venv/`
+- activate venv: `source venv/bin/activate`
+- Install python dependency: `pip install -r requirements.txt`
+
+## Database configuration
+
+Make sure, you have a database configured with proper username/password.
+
+You can either edit `src/config.py` to override the config values for `LocalConfig`.
+
+Or, set environment variables:
+- `DB_HOST`
+- `DB_PORT` (defaults to 5432)
+- `DB_NAME` (defaults to "tempdb" for local config)
+- `DB_USER`
+- `DB_PASSWORD`
+- `DB_TYPE` (defaults to "postgresql")
+- `ITEMS_PER_PAGE` (defaults to 1000)
+- `JWT_SECRET_KEY`
+- `JWT_TOKEN_EXPIRATION_SECONDS` (defaults to 300 seconds)
+
+## Run
+
+Run gunicorn : `gunicorn --bind 0.0.0.0:$PORT --workers=1 --threads=8 src.app:app --timeout=900`
+[make sure to set the port to anything (like: 8000)]
+
+# API Request test
+
+Currently, we provide full CRUD REST api for the catalog tables.
+
+(Note: Following curl examples are for local server. Please, change the endpooints accordingly to point to remote.)
+
+## 1) /catalogue/bulk/csv - POST, Upload CSV
+
+Enables anyone to upload csv of specific format to populate the catalogue table in bulk.
+
+```bash
+curl --location --request POST 'http://127.0.0.1:5000/catalogue/bulk/csv/' \
+--header 'token: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VyIjoidWRheSIsInBhc3N3b3JkIjoidWRheSIsImV4cCI6MTY1MjMzNTgzNn0.UfKN62xWmsXTzwy7tmRbP6I9DbrtXMnidQFFLq6epfs' \
```

Comment:
Is this the actual token used? Maybe remove it and use a placeholder.

Reply:
> Is this the actual token used? Maybe remove it and use a placeholder.

Oops. I missed removing this one.

cc: @ub0005